05-Load-Balancer/versions.tf

```diff
@@ -15,5 +15,5 @@
  */
 
 terraform {
-  required_version = ">= 0.12"
+  required_version = ">= 0.15"
```

Comment:
Any reason why this requires 0.15? We should standardize on one min version.

Reply:
Hi @bharathkkb The integration tests require >= 0.15 for this Lab 05 [output](https://github.com/terraform-google-modules/cloud-foundation-training/pull/48/files#diff-f841f1332ff01dce17165326ae0e0af9b1983e5630a83916b154677792bedcb4R26) due to a change in the lb-http module, however it turns out that Lab 05 itself doesn't make use of that particular output and >=0.13 would be sufficient.  Therefore all the labs would be fine with >= 0.13, but as the integration tests are running >= 0.15, I would suggest it prudent to standardize everything to >= 0.15.  Unless there is a need to support 0.13 and 0.14?

Reply:
@apeabody gotcha, we can just standardize to `>= 0.15`. Most users leverage cloudshell for this tutorial and that also uses >= 0.15.

Reply:
👍